db/schema.rb

```diff
@@ -124,11 +124,10 @@
     t.string "op", limit: 255
     t.uuid "option_node_id"
     t.integer "option_node_old_id"
-    t.uuid "questioning_id"
+    t.uuid "questioning_id", null: false
     t.integer "questioning_old_id"
-    t.uuid "ref_qing_id"
+    t.uuid "ref_qing_id", null: false
     t.integer "ref_qing_old_id"
-    t.string "role", null: false
```

Comment:
I think what's going on here is that your DB has been migrated for one of the condition branches that haven't been merged yet. Can you please remove this from the PR (ideally using rebase to keep the history clean). Then I think most of those test failures will go away.

Reply:
thanks!